docs/TargetMaintenance/CreatingAUnifiedTarget.md

```diff
@@ -55,7 +55,7 @@ These firmware types can be created by building locally and specifying the MCU t
 
 - make sure to enable 'Full chip erase' before flashing, or reset to default with `defaults` in CLI.
 
-- verify that your board is properly reset to defaults: The output of a `diff hardware` in CLI should show `board_name`, `manufacturer_id`, and lines starting with a `#`.
+- verify that your board is properly reset to defaults.
```

Comment:
explain how.

Reply:
It's your board. I'd anticipate you'd do some checks.

Reply:
it should read something like "verify your board is properly reset to defaults by using the cli and running `diff` to show that there are no differences".